lib/tasks/dev/migrate_title_data.rake

```diff
@@ -0,0 +1,28 @@
+
+namespace :dev do
+   
+    desc 'migrate Student.title data to Title.name column and assign reference id'
+
+    task migrate_title_data: :environment do
+
+        # populate Title.name data with unique old_title values 
+        Student.uniq.pluck(:old_title).each do |title|
+            t = Title.find_or_create_by_name(title) unless !title.present?
```

Comment:
unless !title.present?
reads
unless title is not present? ... this is ok
but
this is more readable, if title is present?
`if title.present?`

Reply:
Thanks -  agree that is much clearer!